src/pages/AppLogin.vue

```diff
@@ -71,101 +82,90 @@ onMounted(async () => {
 <template>
   <div>
     <main class="flex">
-      <landing-descriptor />
-      <section class="signin-container">
-        <img
-          src="@/assets/logo.svg"
-          alt="Arcana Logo"
-          style="margin: auto; margin-bottom: 2em"
-          class="laptop-remove"
-        />
-        <div>
-          <h1 style="text-align: center">Welcome To Arcana</h1>
-          <h5
-            style="
-              margin-top: 2vh;
-              margin-right: 2vw;
-              margin-left: 2vw;
-              text-align: center;
-            "
+      <LandingDescriptor />
+      <section class="signin-section">
+        <div class="signin-container">
+          <img
+            src="@/assets/logo.svg"
+            alt="Arcana Logo"
+            class="laptop-remove arcana-logo"
+          />
+          <div>
+            <h1>Welcome</h1>
+            <h5 class="login-description">
+              We’ll email you a login link for a password-free sign in.
+            </h5>
+          </div>
+          <form
+            class="passwordless-container flex column flex-center"
+            @submit.prevent="launchLogin('passwordless')"
           >
-            Let’s get started
-          </h5>
-        </div>
-        <section style="margin-top: 6vh">
-          <h3 style="padding: 1em">Continue With:</h3>
-          <div class="flex column" style="margin-top: 4vh">
-            <div
-              class="flex wrap"
-              style="gap: 1em; justify-content: space-between"
-            >
-              <v-card-button
-                class="sso-button"
+            <VTextField
+              v-model.trim="email"
+              label="Email"
+              placeholder="Enter your email"
+              class="passwordless-email"
```

Comment:
looks like the styles from this class is not applied

Reply:
Styles are getting applied